### PR TITLE
Added titles and text to make plot_orb.py example more clear

### PR DESCRIPTION
**Pull request by rovitotv**
## Description

Added titles and text to make plot_orb.py example more clear

Updated the example for plot_orb.py to address issue #1967.  Issue #1967 calls out plot_censure.py but why not plot_orb.py as well?

### CODE REVIEW

File: doc/examples/features_detection/plot_orb.py [hunk 1]
```diff
@@ -49,8 +49,17 @@
...
 
 plot_matches(ax[1], img1, img3, keypoints1, keypoints3, matches13)
 ax[1].axis('off')
+ax[1].set_title("Original Image VS Transformed Image")
+ax[1].text(1030, 15, 'AffineTransform', ha='left', va='center', color='yellow')
+ax[1].text(1050, 60, 'scaleX = 1.3', ha='left', va='center', color='yellow')
+ax[1].text(1050, 105, 'scaleY = 1.1', ha='left', va='center', color='yellow')
+ax[1].text(1050, 150, 'rotation = 28.65 $^\circ$', ha='left', va='center', color='yellow')
+ax[1].text(1050, 195, 'translateX = 0', ha='left', va='center', color='yellow')
+ax[1].text(1050, 240, 'translateY = -200', ha='left', va='center', color='yellow')
```
Comment (soupault) [inline doc/examples/features_detection/plot_orb.py]: Same as https://github.com/scikit-image/scikit-image/pull/2191/files#r71079425

File: doc/examples/features_detection/plot_orb.py [hunk 2]
```diff
@@ -49,8 +49,17 @@
 
 plot_matches(ax[0], img1, img2, keypoints1, keypoints2, matches12)
 ax[0].axis('off')
+ax[0].set_title("Original Image VS Transformed Image")
+print('Transformed Image using Rotation = 180 degrees')
 
 plot_matches(ax[1], img1, img3, keypoints1, keypoints3, matches13)
 ax[1].axis('off')
+ax[1].set_title("Original Image VS Transformed Image")
+print('Transformed Image using AffineTransform parameters:')
```
Comment (ahojnnes) [inline doc/examples/features_detection/plot_orb.py]: In my opinion, this logging information does not add value to the example... @scikit-image/core Can a second reviewer please contribute their thoughts?

File: doc/examples/features_detection/plot_orb.py [hunk 3]
```diff
@@ -49,8 +49,17 @@
 
 plot_matches(ax[0], img1, img2, keypoints1, keypoints2, matches12)
 ax[0].axis('off')
+ax[0].set_title("Original Image VS Transformed Image")
```
Comment (emmanuelle) [inline doc/examples/features_detection/plot_orb.py]: Please use the more common abbreviation "vs."

Comment (emmanuelle) [inline doc/examples/features_detection/plot_orb.py]: I would instead draw the attention to the detected features, like "ORB features detected in original and transformed image".

**Comment by emmanuelle** [comment]
I don't have a strong opinion about the printed information, but since @rovitotv felt it was needed, I'd say why not?

**Pull request merged**